R/utils-render.R

```diff
@@ -112,8 +112,9 @@ startTemplatedRender <- function(reportJson, author){
   
   #adding appropriately formatted start and end dates to reportJSON
   if (anyDataExist(reportJson[['reportMetadata']][['startDate']]) && anyDataExist(reportJson[['reportMetadata']][['endDate']])) {
-    templateData[['reportMetadata']][['displayStartDate']] <- format(as.Date(fetchReportMetadataField(reportJson,'startDate')), "%Y-%m-%d")
-    templateData[['reportMetadata']][['displayEndDate']] <- format(as.Date(fetchReportMetadataField(reportJson,'endDate')), "%Y-%m-%d")
+    templateData[['reportMetadata']][['periodDisplay']] <- paste(format(as.Date(fetchReportMetadataField(reportJson,'startDate')), "%Y-%m-%d")," to ", format(as.Date(fetchReportMetadataField(reportJson,'endDate')), "%Y-%m-%d"))
+  } else {
```

Comment:
This logic may need to change in the future if we ever have another report that also doesn't include a time period but I don't see that happening any time soon so I think this is good.